cypress/integration/html-reader/errors.ts

```diff
@@ -17,25 +17,23 @@ describe('error states', () => {
       we use visit here instead of 'loadPage' because 'loadPage'
       checks for an iframe that should fail to render in this test 
     */
-    cy.visit('/test/unparsable-manifest');
+    cy.visit('/html/test/unparsable-manifest');
 
     cy.findByRole('heading', { name: 'An error occurred' });
     cy.findByRole('alert', {
-      name: `Network Error: Unparsable JSON file found at ${
-        Cypress.config().baseUrl
-      }/samples/test/unparsable-manifest.json.`,
+      name: /Network Error: Unparsable JSON file found at http.+\/unparsable-manifest.json/i,
     });
   });
 
   // @TODO - Improve this error UX. Requires change in R2D2BC.
   it('throws error for missing resource', () => {
-    cy.visit('/test/missing-resource');
+    cy.visit('/html/test/missing-resource');
 
     cy.getIframeBody().contains('Page not found.', { timeout: 10000 });
   });
 
   it('throws error for missing injectable', () => {
-    cy.visit('/test/missing-injectable');
+    cy.visit('/html/test/missing-injectable');
 
     cy.findByRole('heading', { name: 'An error occurred' });
```

Comment:
We probably don't need this since we will always have default injectables right?